THIRD-PARTY-NOTICES.txt

```diff
@@ -0,0 +1,35 @@
+.NET Core uses third-party libraries or other resources that may be
+distributed under licenses different than the .NET Core software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention. Post an issue or email us:
+
+           dotnet@microsoft.com
+
+The attached notices are provided for information only.
+
+License notice for idevelop/ascii-camera
+------------------------------------
```

Comment:
I cannot even begin to describe to you how /r/mildlyinfuriating this dash line is!

Reply:
Haha, it's the same length as the ones in ASPNETCORE!

Reply:
Well it's not my doing. I literally used this literal code that I wrote to generate the initial ones!

https://github.com/Eilon/ThirdPartyNoticesGenerator/blob/master/Program.cs#L40-L41